CSS/components/alert.css

```diff
@@ -0,0 +1,28 @@
+.avatar{
+    border-radius: 100%;
+    min-width: 5rem;
+    min-height: 5rem;
+    cursor: pointer;
+}
+.avatar-l{
+    min-width: 6rem;
+    min-height: 6rem;
+}
```

Comment:
Class naming could have been better, I suppose this is a large avatar so .avatar-lg sounds more meaningful